File: src/main/java/com/ice/Hello.java
package com.ice;



public class Hello {
    public static void main (String[] args){

//        System.out.println("Hello world");
        String s = "abcde";
        System.out.println(s.charAt(3));
        System.out.println(s.substring(1,4));
        Person p = new Person("ice", 63.6f, 1.8f);
        p.Hello();
        System.out.println(p.bmi());
        int score = 88;
        System.out.println(score < 80  || score > 90);
        char c = 'A';
        System.out.println(c > 'a');
        /*int age = 27;
        Integer age2 = 27;
        char c = '我';
        Character c2 = 'A';
        byte b = 120;
        float weight = 63.3f;
        float height = 1.7f;
        boolean adult = true;
        boolean enroll = false;
        String name = "ice";*/
    }
}

File: src/main/kotlin/com/ice/Hello.kt
package com.ice

import com.ice.kotlin.Student

fun main() {
    val student = Student ("ice", 60, 60)

    var s:String? = "abcde"
    s = null
    println(s?.length)
    println(s?.get(3))
    println(s?.substring(3))
    val h = Human(weight = 63.8f, height = 1.8f)
    println(h.bmi())
    val score = 88
    println(score > 80)
    val c : Char = 'A'
    println(c.toInt() > 60)
    /*var age = 19
    age = 20
    var weight  = 66.3
    var name : String
    name = "ice"*/
}

class Human(var name: String = "", var weight : Float, var height : Float) {
    init {
        println("test $weight")
    }
//    constructor(name : String, weight : Float, height :Float) : this(weight, height)
    fun bmi() : Float {
        val bmi = weight / (height * height)
        return bmi
    }

    fun hello(){
        println("Hello kotlin")
    }
    }